### ISSUE TITLE
[BUG] I want to add transition to mobile burger menu 

### ISSUE THREAD

**Issue by ratishjain12** [ArslanYM/StarterHive#316]
### Description of the bug

Addition of transition in mobile burger menu when it opens and closes

Current state - it is just popping up

@ArslanYM assign me this issue to I can work on it

### Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ArslanYM/StarterHive/blob/main/CONTRIBUTING.md)
- [X] I have checked the existing [issues](https://github.com/ArslanYM/StarterHive/issues)
- [X] I am willing to work on this issue (optional)

**Comment by ArslanYM** [comment]
Can you provide any examples

**Comment by ratishjain12** [comment]
@ArslanYM  something like this 

https://github.com/ArslanYM/StarterHive/assets/43003421/102f76f5-f4d6-446a-96aa-3a713b84f92a

If this works ping me I'll open a PR

**Comment by ArslanYM** [comment]
I dont think we need something like this rn ,  but we surely need some help in setting a good color scheme for the light mode of the project. Do you think you can work on it?

**Comment by ratishjain12** [comment]
Yeah I also didn't like the grey colour navbar tabs I think the text color should be little darker than gray

We discuss this on how you want the page to look like do let me know.

**Comment by inbuilt-aura** [comment]
@ratishjain12  Hey, is this issue open.If yes, i want to work on this issue.
